simcase-game/src/views/Ranking.js

```diff
@@ -61,6 +51,12 @@ const name = {
   fontWeight: "bold"
 };
 
+const you = {
+  fontSize: "20px",
+  fontWeight: "bold",
+  color: "red"
+};
+
```

Comment:
Could be better like this:
```jsx
const font = {
  fontSize: "20px",
  fontWeight: "bold"
};

const you = {...font, color: "red"};
```

Reply:
Thanks you, I forget you are using variables to styles. Going to change this.